src/v2/options.ts

```diff
@@ -239,6 +244,15 @@ export function getGlobalOptions(): GlobalOptions {
   return globalOptions || {};
 }
 
+/**
+ * Options that can be set on an individual HTTPS Cloud Function.
+ */
+export interface HttpsOptions extends Omit<GlobalOptions, 'region'> {
```

Comment:
What do we think about also moving event handler options here and exporting them from their path?

Reply:
Yes - more breaking changes for v2 users before IO!

Reply:
I think it should probably stay in https because it lines up with the fact that PubSubOptions are, for example, in pubsub.ts